contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -356,12 +352,7 @@ contract LoanTokenLogicStandard is AdvancedToken {
         view
```

Comment:
`tokenPrice` function is public which can be called internally and externally. why having extra _tokenPrice?